src/huak/env/venv.rs

```diff
@@ -64,19 +65,19 @@ impl Venv {
         let from = match self.path.parent() {
             Some(p) => p,
             _ => {
-                return Err(CliError::new(
-                    anyhow::format_err!("invalid venv path"),
-                    2,
-                ))
+                return Err(CliError::from(anyhow::format_err!(
+                    "Invalid venv path"
+                )))
             }
         };
 
         let name = self.name()?;
         let args = ["-m", "venv", name];
 
-        crate::utils::command::run_command("python", &args, from)?;
-
-        Ok(())
+        match crate::utils::command::run_command("python", &args, from) {
+            Ok(_) => Ok(()),
+            Err(_) => Err(CliError::new(CliErrorType::PythonNotFound)),
```

Comment:
Unfortunately we'd need to do more than this here to determine if the error is because there was no python alias. We can do this in a follow up PR.

Tag #160

Reply:
Yup, thats the plan